features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/mssql.sql

```diff
@@ -1471,9 +1471,9 @@ CREATE TABLE IDN_NOTIFICATION_ORG_TEMPLATE (
     ID INTEGER IDENTITY,
     TEMPLATE_KEY VARCHAR(50) NOT NULL,
     LOCALE VARCHAR(50) NOT NULL,
-    SUBJECT VARCHAR(4000),
-    BODY TEXT,
-    FOOTER TEXT,
+    SUBJECT NVARCHAR(4000),
+    BODY NVARCHAR(MAX),
```

Comment:
Cannot we consider `VARBINARY` here?

Reply:
`VARBINARY ` is used to store binary data.. Here we are handling unicode text.. While we can technically use `VARBINARY` to store this data in the binary format, wouldn't it better to use more native datatype which doesn't require additional binary-text conversion?

Also, can you share what's the problem we can solve, if we are to changing the datatype from `NVARCHAR` to `VARBINARY`?

Reply:
You're correct; if we only need to support Unicode text (HTML codes), then using `NVARCHAR` should be sufficient.